Add home section

RiaDhanani opened the pull request:
### Description

Developed home section according to given styling requirements.

Fixes #111 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
![#111](https://user-images.githubusercontent.com/48614937/89265277-dc9a0d80-d651-11ea-9742-66fcde804208.png)



### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials


**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

annabauza commented on src/Components/Home/index.js:
```diff
@@ -0,0 +1,73 @@
+function Home() {
+    return (
+        <View style={styles.home}>
+        <Text style={styles.heading}>INTERESTED IN COMPUTING</Text>
```
we need to keep it as a section not in home. Please reuse Section Headers like wiki advice

RiaDhanani replied:
Fixed it now. Can you check?

RiaDhanani commented:
I renamed the component to ContributionSection so that it is clear that it is just  a section in the home page. Then maybe when the home page is being developed we can add index.js.

RiaDhanani commented:
@vj-codes @annabauza Could you review the changes?

19-pragya commented on src/Components/Home/ContributionSection.js:
```diff
@@ -0,0 +1,75 @@
+    }
+    
+    
+    export default ContributionSection;
```
you can remove this {export default ContributionSection as it is of no use after  the program is written.

RiaDhanani replied:
But later we would need to use it right so why keep it as a small task  to add export again?

annabauza replied:
this export is good @19-pragya

annabauza commented on src/Components/Home/ContributionSection.js:
```diff
@@ -0,0 +1,75 @@
+    return (
+        <View>
+        <SectionHeader title= "INTERESTED IN COMPUTING" />
+        <Text style={styles.description}>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed</Text>
```
Lorem ipsum will not be accepted.

RiaDhanani replied:
@annabauza Lorem ipsum was given in the design. I can change it but what should I change it to?

vj-codes replied:
@annabauza @RiaDhanani 'You can start contributing to AnitaB.org Open Source through one or more fields of your choice' hows this?

annabauza replied:
All texts should be held in separate files. Grid should not be created entirely manually it should be 6 same components. Why repead code? On screanshots texts feel shifted not centred to the images. Also I can't tell from screashot how is responsiveness of this implementation.

Sonal240 replied:
Is the content issue yet created for this section? If yes is the pr raised for it? @vj-codes

Sonal240 commented:
> @Sonal240 @sakshi1499 @nandini45 @annabauza wasnt the title supposed to be '**Interested in contributing**' and not computing?Interested in computing makes no sense here

yep absolutely

Sonal240 commented:
add a video showing both responsive and web view @RiaDhanani once you are done with the requested changes

RiaDhanani commented:
> add a video showing both responsive and web view @RiaDhanani once you are done with the requested changes

![contribution](https://user-images.githubusercontent.com/48614937/91753222-348e4a80-ebe5-11ea-838f-581bfd7901d6.gif)
I've removed the duplication and  added the gif to show responsiveness. Please review the changes.
@annabauza @Sonal240 @vj-codes

annabauza commented on src/Components/Home/contribution/ContributionSection.js:
```diff
@@ -0,0 +1,56 @@
+
+
+function ContributionSection() {
+    const contributionCards = [
```
can we move text and pngs to content folder ?

RiaDhanani replied:
I can shift the text to the content folder , just wanted to understand why are we keeping all text in a separate folder? And all the other images are in the assets folder so should I still move it to content?

annabauza replied:
So we can easily update content, without looking throw whole code

annabauza commented on src/Components/Home/contribution/ContributionSection.js:
```diff
@@ -0,0 +1,56 @@
+        <View>
+        <SectionHeader title= "INTERESTED IN CONTRIBUTING?" />
+        <Text style={styles.description}>You can start contributing through one or more fields of your choice</Text>
+        <View style={styles.container}>
```
if you do here `flexDirection:'row'` and `flexWrap:'wrap'` and  `justifyContent: 'space-between', or  justifyContent: 'space-around',` you would not have to have separate style for each card.

RiaDhanani replied:
I tried doing it with justifyContent and flex but since I need to maintain the left and right margin as 32, the design alignment shifts. Anything else I should try?

annabauza replied:
well then apply padding to container above.

RiaDhanani replied:
@annabauza I've tried eliminating the styling for individual cards using the flex properties and padding as you suggested but it disrupts the alignment , especially the margins.  When I add the padding, the left margin increases.
![anitab-org](https://user-images.githubusercontent.com/48614937/93751941-8beb6d80-fc1b-11ea-89a7-0ede43a5ddf4.png)

annabauza reviewed (changes requested):
small changes required

annabauza commented:
@RiaDhanani please follow the pr deployment from wiki so qa team could review

RiaDhanani commented:
> @RiaDhanani please follow the pr deployment from wiki so qa team could review

Couldn't find the pr deployment in the wiki page. Could you send me a link?

nandini45 commented:
https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR
 i think this is the link @RiaDhanani
